start-dev.sh

```diff
@@ -4,7 +4,7 @@ docker build -t ipfs/jenkins:latest .
 # TODO detect if /var/run/docker.sock exists and errors if it doesn't
 # TODO allow people to change path to socket
 docker run -p 8090:8080 \
-	--privileged \
 	-v /var/run/docker.sock:/var/run/docker.sock \
+	--group-add "$(getent group docker | cut -d':' -f 3)" \
```

Comment:
It adds main process to supplementary group (with ID of the docker group from the host).
Just using `docker` doesn't work as the GID will be different in container than outside of it so we use GID instead of name.